src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -543,13 +543,30 @@ public FeedbackResponse getFeedbackResponseEntityOptimized (FeedbackResponseAttr
     }
     
     /**
+     * Updates the feedback response identified by {@code newAttributes.getId()} and 
+     *   changes the {@code updatedAt} timestamp to be the time of update.
+     * For the remaining parameters, the existing value is preserved 
+     *   if the parameter is null (due to 'keep existing' policy).<br> 
+     * Preconditions: <br>
+     * * {@code newAttributes.getId()} is non-null and correspond to an existing feedback response.
+     * @throws EntityDoesNotExistException 
+     * @throws InvalidParametersException 
+     */
+    public void updateFeedbackResponse(FeedbackResponseAttributes newAttributes)
+        throws InvalidParametersException, EntityDoesNotExistException {
```

Comment:
I only followed the previous styling (i.e. second line indented two spaces) as in the following line.

Previously, the text <code>if the parameter is null ...</code> on line 549 is indented that way and I follow that styling for line 547.